docs/tutorials/getting_started.md

```diff
@@ -167,42 +167,43 @@ Model   Controller          Cloud/Region        Version  SLA          Timestamp
+smf                       1.6.2    active       1  sdcore-smf-k8s            1.6/edge       764  10.152.183.220  no       
+traefik                   2.11.0   waiting      1  traefik-k8s               latest/stable  203  10.152.183.27   no       installing agent
+udm                       1.5.1    active       1  sdcore-udm-k8s            1.6/edge       625  10.152.183.228  no       
+udr                       1.6.1    active       1  sdcore-udr-k8s            1.6/edge       612  10.152.183.141  no       
+upf                       1.4.0    active       1  sdcore-upf-k8s            1.6/edge       678  10.152.183.229  no       
 
 Unit                         Workload  Agent  Address       Ports  Message
-amf/0*                       active    idle   10.1.213.193
-ausf/0*                      active    idle   10.1.213.236
-grafana-agent/0*             blocked   idle   10.1.213.248         logging-consumer: off, grafana-cloud-config: off
-mongodb/0*                   active    idle   10.1.213.252         Primary
-nms/0*                       active    idle   10.1.213.214
-nrf/0*                       active    idle   10.1.213.195
-nssf/0*                      active    idle   10.1.213.243
-pcf/0*                       active    idle   10.1.213.225
-router/0*                    active    idle   10.1.213.231
-self-signed-certificates/0*  active    idle   10.1.213.232
-smf/0*                       active    idle   10.1.213.229
-traefik/0*                   error     idle   10.1.213.208         hook failed: "ingress-relation-created"
-udm/0*                       active    idle   10.1.213.203
-udr/0*                       active    idle   10.1.213.250
-upf/0*                       active    idle   10.1.213.200
-
-Offer  Application  Charm           Rev  Connected  Endpoint  Interface  Role
-amf    amf          sdcore-amf-k8s  834  0/0        fiveg-n2  fiveg_n2   provider
-upf    upf          sdcore-upf-k8s  685  0/0        fiveg_n3  fiveg_n3   provider
+amf/0*                       active    idle   10.1.194.236         
+ausf/0*                      active    idle   10.1.194.243         
+grafana-agent/0*             blocked   idle   10.1.194.196         Missing ['grafana-cloud-config']|['logging-consumer'] for logging-provider; ['grafana-cloud-config']|['send-remote-wr...
+mongodb/0*                   active    idle   10.1.194.249         
+nms/0*                       active    idle   10.1.194.203         
+nrf/0*                       active    idle   10.1.194.247         
+nssf/0*                      active    idle   10.1.194.227         
+pcf/0*                       active    idle   10.1.194.224         
+router/0*                    active    idle   10.1.194.245         
+self-signed-certificates/0*  active    idle   10.1.194.225         
+smf/0*                       active    idle   10.1.194.255         
+traefik/0*                   error     idle   10.1.194.246         hook failed: "certificates-relation-changed"
```

Comment:
In the first deployment, Traefik fails with ` hook failed: "ingress-relation-created"` not `certificates-relation-changed`.

Reply:
You mean the `ingress-relation-created` looks better? :) 
From the functional perspective it doesn't matter on which event does the Traefik fail. In both cases the reason is the same - the LB IP address is not a valid URL.